### DISCUSSION TITLE
Welcome to the Nhost Community!

### DISCUSSION THREAD

**Discussion by elitan** [nhost/nhost#2] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we build together 💪.

To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by lumi-mustafa-hanif** [comment]
Hello,

I want to know where is the main discussion forum? I am planning to create a production website using this.

**Reply by jmarbutt** [reply]
There is also a lot of discussion on the [discord server](https://discord.com/invite/9V7Qb2U)

**Reply by elitan** [reply]
@mustafa-hanif here on GitHub is also fine.

**Comment by elitan** [comment]
You found the right  place :)

On Sun, 28 Mar 2021 at 09:50, Mustafa Hanif ***@***.***>
wrote:

> Hello,
>
> I want to know where is the main discussion forum? I am planning to create
> a production website using this.
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/nhost/nhost/discussions/2#discussioncomment-540002>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/AACRAKSYHREF4TLVQXPJSXTTF3NWFANCNFSM4XUOU74Q>
> .
>
-- 
---
Johan Eliasson
Founder / CEO | Nhost <https://nhost.io/>
Linkedin <https://www.linkedin.com/in/johan--eliasson/> / Twitter
<https://twitter.com/elitasson>
Stockholm, Sweden

Book a call with me <https://calendly.com/nhost/johan?month=2021-01>

**Reply by lumi-mustafa-hanif** [reply]
I am unable to use the `API functions` feature, and what are the limitations of it, I am coming from Vercel where it just works.

**Reply by elitan** [reply]
What's the error you're getting? Maybe you can create a separate issue here: https://github.com/nhost/nhost/issues/new ?

**Comment by PtradeLLC** [comment]


**Reply by dbarrosop** [reply]
Thanks and welcome, would you mind opening a new discussion topic in the Q&A section?

You added your question as a comment to a Welcome message for the discussions forum :)

**Reply by PtradeLLC** [reply]
Sorry about that. I was trying to find the Q&A section, but resolved to posting on here. I'll do that right away.
